### PR TITLE
Draft to Normal PR Added

### PR DESCRIPTION
**Pull request by KJStick**
… to actual pr

### CODE REVIEW

File: wxt_cidrbot/git_webhook_handler.py [hunk 1]
```diff
@@ -121,11 +125,15 @@ def webhook_request(self, event):
...
-    def triage_issue(self, installation_id, json_string, x_event_type):
+    def triage_issue(self, installation_id, json_string, x_event_type, converted_from_draft=False):
         event_info = self.check_installation(installation_id)
         issue_type = "Pull request"
         query_key = "pull_request"
 
+        pull_request_action_msg = "created in"
+        if converted_from_draft:
+            pull_request_action_msg = "has been converted from draft in"
+
```
Comment (jlothian) [inline wxt_cidrbot/git_webhook_handler.py:136]: This is probably better written as an if/else - if you need to do some other stuff later on based on the conditions, you won't have to re-do any of the logic flow.

File: wxt_cidrbot/git_webhook_handler.py [hunk 2]
```diff
@@ -497,3 +501,13 @@ def get_approved_reviews(self, json_string, headers):
         approved_reviewers = approved_reviewers[:-2]
 
         return {'approved_reviews': approved_reviews, 'approved_reviewers': approved_reviewers}
+
+    def get_code_owners_count(self, reviewers: tuple) -> int:
+        """returns the number of code owners"""
+        num_code_owners = 0
+
+        for reviewer in reviewers:
```
Comment (jlothian) [inline wxt_cidrbot/git_webhook_handler.py:509]: can't this just be `len(reviewers)`?

**Review by jlothian — CHANGES_REQUESTED** [review]

**Review by jlothian — APPROVED** [review]
:+1:

**Review by ppajersk — APPROVED** [review]
+1

**Pull request merged**